Mathlib/Topology/UniformSpace/UniformConvergenceTopology.lean

```diff
@@ -792,6 +792,24 @@ def uniformEquivUniformFun (h : univ ∈ 𝔖) : (α →ᵤ[𝔖] β) ≃ᵤ (α
     filter_upwards [UniformOnFun.gen_mem_uniformity _ _ h hU] with f hf x using hf x (mem_univ _)
   uniformContinuous_invFun := uniformContinuous_ofUniformFun _ _
 
+/-- If `𝔖` and `𝔗` are families of sets in `α`, then the identity map
+`(α →ᵤ[𝔗] β) → (α →ᵤ[𝔖] β)` is uniformly continuous if every `s ∈ 𝔖` is containined in a finite
+union of elements of `𝔗`.
+
+With more API around `Order.Ideal`, this could be phrased in that language instead. -/
```

Comment:
I'm a bit annoyed at this proof being so long, but indeed there's not much point making it cleaner until we have a proper API for order ideals, so I think this is fine